data/uvepss/config.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- University of Victory Endings Project Static Site configuration -->
+<!-- See, e.g., https://endings.uvic.ca/staticSearch/docs/howDoIUseIt.html -->
+<config xmlns="http://hcmc.uvic.ca/ns/staticSearch" version="1">
+  <params>
+    <searchFile>../../vol/search.html</searchFile>
+    <recurse>true</recurse>
+    <stopwordsFile>stopwords_en.txt</stopwordsFile>
+    <dictionaryFile>words_en.txt</dictionaryFile>
+    <!-- above required; below optional -->
+    <maxKwicsToHarvest>12</maxKwicsToHarvest> <!-- ignored because phrasalSearch is true -->
+    <maxKwicsToShow>5</maxKwicsToShow>
+    <versionFile>version.txt</versionFile>
+    <createContexts>true</createContexts>
+    <phrasalSearch>true</phrasalSearch>
+    <scoringAlgorithm>raw</scoringAlgorithm> <!-- using "tf-idf" gave long values < 0 which folks did not find very helpful -->
+    <totalKwicLength>11</totalKwicLength>
+    <kwicTruncateString>…</kwicTruncateString>
+    <verbose>false</verbose> 	<!-- comment out for sydb_uvepss_02, but must leave uncommented for 1.4.5;  turn on for debugging index process -->
+    <indentJSON>true</indentJSON> <!-- turn on for debugging -->
+    <outputFolder>uvepss</outputFolder>
+  </params>
+  <rules>
+    <rule match="/html/body//text()[not( ancestor::div[@id eq 'mainContent'] )]" weight="0"/>
```

Comment:
I am not sure what you mean, here. Do you mean

- avoiding HTML files that are ill-formed XML, 
- avoiding some HTML files and indexing others, or
- avoiding portion of an HTML file and indexing the rest?

The answer to the first is “no”. An ill-formed file will not even be read, so it cannot be parsed for its `<meta>` elements.

As for the second, yes, insofar as we can insert `<meta name="indexMe" content="no"/>` (or whatever) into the well-formed HTML files we do not want indexed. Then either `<rule match="//*[ancestor::HTML/head/meta[ @name eq 'indexMe'  and  @content='no']]" weight="0"/>` or `<exclude type="filter" match="meta[ @name eq 'indexMe'  and  @content='no']"/>` would suppress indexing.

As for the third, I am not aware of any StaticSearch functionality that would allow us to suppress indexing of some portion an HTML document based on its `<meta>` elements. (But since that is what `<rule>` with a `@weight` of "0" does, I am not sure why we would want to use `<meta>` for this, anyway.)

This particular rule asks Static Search to avoid indexing the top navigation, the sidebars, etc. — ignore everything that is _not_ in "mainContent".